librad/src/git/storage.rs

```diff
@@ -992,6 +992,16 @@ where
 
         Ok(())
     }
+
+    // TODO(nuno): quoting Kim: [this] is a bit too optimistic I think
+    // -- git2 unifies everything into a single Error enum, but we only
+    // want to return false if the ErrorCode is NotFound, and otherwise
```

Comment:
Done. I felt a bit resistant in doing this since it adds a lot of bloat while the bubbled up error is not fully propagated, but it is sensible to return `Result<bool, Error>` anyways.

Reply:
Strictly better than swallowing all errors :)

Reply:
Yes indeed :)